### DISCUSSION TITLE
Ошибка

### DISCUSSION THREAD

**Discussion by salindev** [MrZillaGold/VK2Discord#266] [Q&A] [answered]
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/C:/Users/User/Desktop/VK2Discord-master/dist/index.js' imported from C:\Users\User\Desktop\VK2Discord-master\
![image](https://user-images.githubusercontent.com/79220416/130018435-1b53dcf2-11f8-4020-8c8e-3e6b5a1d6bf7.png)

**Comment by egorprnn** [comment, marked as answer]
Используйте версии из [релизов](https://github.com/MrZillaGold/VK2Discord/releases/latest), а не из корня репозитория.
Также можете самостоятельно собрать проект используя `npm run-script build`
